src/zui/ZUIAccessList/index.tsx

```diff
@@ -0,0 +1,117 @@
+import { FC } from 'react';
+import { Divider, FormControl, List, MenuItem, Select } from '@mui/material';
+import { FormattedMessage, useIntl } from 'react-intl';
+
+import AccessListItem from './AccessListItem';
+import { ZetkinObjectAccess } from 'core/api/types';
+import { ZetkinOfficial } from 'utils/types/zetkin';
+import ZUIRelativeTime from 'zui/ZUIRelativeTime';
+
+interface ZUIAccessListProps {
+  accessList: ZetkinObjectAccess[];
+  officials: ZetkinOfficial[];
+  onChangeLevel?: (
+    personId: number,
+    level: ZetkinObjectAccess['level']
+  ) => void;
+  onRevoke?: (personId: number) => void;
+  orgId: number;
+}
+
+const ZUIAccessList: FC<ZUIAccessListProps> = ({
+  accessList,
+  officials,
+  onChangeLevel,
+  onRevoke,
+  orgId,
+}) => {
+  const intl = useIntl();
+  let first = true;
+  return (
+    <List>
+      {officials.map((item) => {
+        const showDivider = !first;
+        first = false;
```

Comment:
This is to decide whether to show the divider (between items) or not. If there are N items, there should be N-1 dividers, e.g. if there are 4 items, there should be 3 dividers (1 divider 2 divider 3 divider 4).

This means that I can't add a divider for every iteration in the loop. I opted to add the dividers BEFORE each item, except for the first item, where I don't add a divider.

There are of course many ways of doing this, but I do it by having a variable `first` that is only true for the first item. I set it to `true` before I start looping, and then during the first iteration of the loop I set it to `false`, so that it's `false` for all subsequent iterations.

Does that make sense? Let me know if you have a better idea on how to solve this. It felt a little odd, but also made sense. I thought about creating a reusable component for it, but ended up doing it this way instead.

Reply:
Duh! Your solution is much better. I really felt like I was missing something obvious, and there it is. I will change it. 😊

Reply:
Hey! It's me again, back to change my mind again. I can't just use index, because there are _two_ lists. The first one may or may not be empty. I remember now, that's why I decided to do it this way. Let's keep it. 😊